dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/OneAgentMetadataEnricher.java

```diff
@@ -106,8 +97,7 @@ String getIndirectionFilename(Reader fileContents, String indirectionBaseName)
       // read file line by line, and stop if the end of the file is reached.
       while ((line = reader.readLine()) != null) {
```

Comment:
There should only be one file name in there anyway with no leading empty lines so I'd only check the first line and null if there's none.